packages/types/src/discord/guild.ts

```diff
@@ -3,33 +3,67 @@
 import type { ChannelTypes, DiscordChannel, DiscordOverwrite, DiscordThreadMember, SortOrderTypes } from './channel.js'
 import type { DiscordEmoji } from './emoji.js'
 import type { DiscordGuildCreateExtra } from './gateway.js'
+import type { DiscordInvite } from './invite.js'
 import type { OAuth2Scope } from './oauth2.js'
 import type { DiscordRole } from './permissions.js'
 import type { DiscordSticker } from './sticker.js'
 import type { DiscordAvatarDecorationData, DiscordUser } from './user.js'
 
 /** https://discord.com/developers/docs/resources/guild#guild-object-guild-structure */
 export interface DiscordGuild extends Partial<DiscordGuildCreateExtra> {
+  /** Guild id */
+  id: string
   /** Guild name (2-100 characters, excluding trailing and leading whitespace) */
   name: string
+  /** Icon hash */
+  icon: string | null
+  /** Icon hash, returned when in the template object */
+  icon_hash?: string | null
+  /** Splash hash */
+  splash: string | null
+  /** Discovery splash hash; only present for guilds with the "DISCOVERABLE" feature */
+  discovery_splash: string | null
   /** True if the user is the owner of the guild */
   owner?: boolean
```

Comment:
https://discord.com/developers/docs/change-log#whats-changing

> Existing Guilds owned by bots will have their ownership transferred to a real user

The docs hasn't updated yet. Should we keep it as-is or add a `@deprecated` note?

Also: 

https://discord.com/developers/docs/resources/guild#guild-object-guild-structure

> These fields are only sent when using the [GET Current User Guilds](https://discord.com/developers/docs/resources/user#get-current-user-guilds) endpoint and are relative to the requested user

Should we add this note?

```suggestion
/**
 * True if the user is the owner of the guild
 * @remarks This field is only sent when using the {@link getGuilds | getGuilds} helper
 */
owner?: boolean
```

Reply:
> The docs hasn't updated yet. Should we keep it as-is or add a @deprecated note?
`owner` is stil used for OAuth2 so we need to keep it, we can add the node however